src/generic_ui/locales/en/messages.json

```diff
@@ -998,7 +998,7 @@
   },
   "WHAT_IS_DIGITAL_OCEAN_ANSWER": {
     "description": "",
-    "message": "<p>DigitalOcean is a cloud hosting provider. Once you create an account and set up a payment method, uProxy will be able to create a server under your account. By default, uProxy will set up a server that costs $10 per month. (DigitalOcean offers servers starting at $5/month, and we plan to support these in the future, but for usable performance, the next level up is currently required.) In addition to credit cards, DigitalOcean accepts PayPal, which integrates with a number of payment systems, including UnionPay. You can pause your server or terminate your DigitalOcean account at any time.</p>"
+    "message": "<p>DigitalOcean is a cloud hosting provider. Once you create an account and set up a payment method, uProxy will be able to create a server under your account. By default, uProxy will set up a server that costs $10 per month. (DigitalOcean offers servers starting at $5 per month, and we plan to support these in the future, but for usable performance, the next level up is currently required.) In addition to credit cards, DigitalOcean accepts PayPal, which integrates with a number of payment systems, including UnionPay. You can pause your server or terminate your DigitalOcean account at any time.</p>"
```

Comment:
Yeah, I replace "$5/month" with "$5 per month", which is how we're spelling it in all the other places.